Patch developed by Kurt Lust for LUMI.
Basically, the makefiles are junk as they fix the installation directory
and fix compiler names in a stupid way.
diff -Nau fpocket-4.2.2/makefile.orig fpocket-4.2.2/makefile
diff -Nau fpocket-4.2.2/src/qhull/Makefile.orig fpocket-4.2.2/src/qhull/Makefile

--- fpocket-4.2.2/makefile.orig	2025-04-07 19:58:59.000000000 +0300
+++ fpocket-4.2.2/makefile	2025-04-07 20:20:49.000000000 +0300
@@ -5,6 +5,11 @@
 ARCH	    = LINUXAMD64
 export ARCH
 
+CC = cc
+CCQHULL = $(CC)
+LINKER = LD_LIBRARY_PATH=$(PLUGINDIR)/$(ARCH)/molfile $(CC)
+LINKERQHULL = $(CC)
+
 PLUGINDIR   = plugins
 
 PATH_OBJ    = obj/
@@ -15,11 +20,11 @@
 PATH_QHULL  = src/qhull/src
 PATH_LIB	= $(PLUGINDIR)/$(ARCH)/molfile/
 
-BINDIR  = /usr/local/bin/
-LIBDIR  = /usr/local/lib/
-
-MANDIR  = /usr/local/man/man8/
+PREFIX = /usr/local
 
+BINDIR  = $(PREFIX)/bin/
+LIBDIR  = $(PREFIX)/lib/
+MANDIR  = $(PREFIX)/share/man/man8/
 
 FPOCKET     = fpocket
 TPOCKET	    = tpocket
@@ -30,17 +35,6 @@
 PROGALL		= $(PATH_BIN)$(FPOCKET) $(PATH_BIN)$(TPOCKET) $(PATH_BIN)$(DPOCKET) $(PATH_BIN)$(MDPOCKET)
 PROGMDPOCKET    = $(PATH_BIN)$(MDPOCKET)
 
-ifeq ($(CXX),g++)
-  CC          = gcc
-  CCQHULL	    = gcc
-  LINKER      = LD_LIBRARY_PATH=$(PLUGINDIR)/$(ARCH)/molfile gcc
-  LINKERQHULL = gcc
-else 
-  CCQHULL	    = clang
-  CC          = clang
-  LINKER      = LD_LIBRARY_PATH=$(PLUGINDIR)/$(ARCH)/molfile clang
-  LINKERQHULL = clang
-endif
 
 CGSL        = -DMD_NOT_USE_GSL -I$(PATH_GSL)include
 COS         = -DM_OS_LINUX
@@ -179,6 +173,7 @@
 
 install:
 	mkdir -p $(BINDIR)
+	mkdir -p $(LIBDIR)
 	mkdir -p $(MANDIR)
 	cp $(PATH_BIN)$(FPOCKET) $(BINDIR)
 	cp $(PATH_BIN)$(TPOCKET) $(BINDIR)
--- fpocket-4.2.2/src/qhull/Makefile.orig	2025-04-07 20:15:39.000000000 +0300
+++ fpocket-4.2.2/src/qhull/Makefile	2025-04-07 20:17:11.000000000 +0300
@@ -6,7 +6,7 @@
 #   For static builds, a simple alternative is src/libqhull_r/Makefile
 #   
 # Variables
-#   DESTDIR        destination directory for 'make install'.
+#   PREFIX        destination directory for 'make install'.
 #   BINDIR         directory where to copy executables
 #   DOCDIR         directory where to copy html documentation
 #   INCDIR         directory where to copy headers
@@ -77,12 +77,12 @@
 # Unix line endings (\n)
 # $Id: //main/2015/qhull/Makefile#10 $
 
-DESTDIR = /usr/local
-BINDIR	= $(DESTDIR)/bin
-INCDIR	= $(DESTDIR)/include
-LIBDIR	= $(DESTDIR)/lib
-DOCDIR	= $(DESTDIR)/share/doc/qhull
-MANDIR	= $(DESTDIR)/share/man/man1
+PREFIX = /usr/local
+BINDIR	= $(PREFIX)/bin
+INCDIR	= $(PREFIX)/include
+LIBDIR	= $(PREFIX)/lib
+DOCDIR	= $(PREFIX)/share/doc/qhull
+MANDIR	= $(PREFIX)/share/man/man1
 
 # if you do not have enscript, try a2ps or just use lpr.  The files are text.
 PRINTMAN = enscript -2rl
@@ -92,13 +92,8 @@
 
 #for Gnu's gcc compiler, -O3 for optimization, -g for debugging, -pg for profiling
 # -fpic needed for gcc x86_64-linux-gnu.  Not needed for mingw
-ifeq ($(CXX),g++)
-  CC          = gcc
-  CXX		  = g++:
-else 
-  CC          = clang
-  CXX       = clang
-endif
+CC          = cc
+CXX		    = CC
 
 
 CC_OPTS1  = -O3 -ansi -Isrc -fpic $(CC_WARNINGS)
